plugin/cpp_auto_include.vim

```diff
@@ -82,32 +82,46 @@ module CppAutoInclude
-    ['cassert',        false, true , R[F['assert']] ],
-    ['cstring',        false, true , R[F['mem(?:cpy|set|n?cmp)', 'str(?:len|n?cmp|n?cpy|error|cat|str|chr)']] ],
-    ['cstdlib',        false, true , R[F['system','abs','ato[if]', 'itoa', 'strto[dflu]+','free','exit','l?abs','s?rand(?:_r|om)?','qsort'], C['EXIT_[A-Z]*', 'NULL']] ],
-    ['cmath',          false, true , R[F['pow[fl]?','a?(?:sin|cos|tan)[hl]*', 'atan2[fl]?', 'exp[m12fl]*', 'fabs[fl]?', 'log[210fl]+', 'nan[fl]?', '(?:ceil|floor)[fl]?', 'l?l?round[fl]?', 'sqrt[fl]?'], C['M_[A-Z24_]*', 'NAN', 'INFINITY', 'HUGE_[A-Z]*']] ],
-    ['strings.h',      false, true , R[F['b(?:cmp|copy|zero)', 'strn?casecmp']] ],
-    ['typeinfo',       false, true , R[C['typeid']] ],
-    ['new',            true , true , R[F['set_new_handler'], C['nothrow']] ],
-    ['limits',         true , true , R[T['numeric_limits']] ],
-    ['algorithm',      true , true , R[F['(?:stable_|partial_)?sort(?:_copy)?', 'unique(?:_copy)?', 'reverse(?:_copy)?', 'nth_element', '(?:lower|upper)_bound', 'binary_search', '(?:prev|next)_permutation', 'min', 'max', 'count', 'random_shuffle', 'swap']] ],
-    ['numeric',        true , true , R[F['partial_sum', 'accumulate', 'adjacent_difference', 'inner_product']] ],
-    ['iostream',       true , true , R[C['c(?:err|out|in)']] ],
-    ['sstream',        true , true , R[C['[io]?stringstream']] ],
-    ['bitset',         true , true , R[T['bitset']] ],
-    ['complex',        true , true , R[T['complex']] ],
-    ['deque',          true , true , R[T['deque']] ],
-    ['queue',          true , true , R[T['queue','priority_queue']] ],
-    ['list',           true , true , R[T['list']] ],
-    ['map',            true , true , R[T['(?:multi)?map']] ],
-    ['set',            true , true , R[T['(?:multi)?set']] ],
-    ['vector',         true , true , R[T['vector']] ],
-    ['iomanip',        true , true , R[F['setprecision', 'setbase', 'setw'], C['fixed', 'hex']]],
-    ['fstream',        true , true , R[T['fstream']] ],
-    ['ctime',          false, true , R[F['time', 'clock'], C['CLOCKS_PER_SEC']]],
-    ['string',         true , true , R[C['string']] ],
-    ['utility',        true , true , R[T['pair'], F['make_pair']] ],
+    ['algorithm'     , true , true , R[F['move(?:_backward)?', 'find(?:_(?:(?:if(?:_not)?)|end|(?:first_of))?)?', '(?:stable_|partial_)?sort(?:_copy)?', 'unique(?:_copy)?', 'reverse(?:_copy)?', 'nth_element', '(?:lower|upper)_bound', 'binary_search', '(?:prev|next)_permutation', 'min', 'max', 'count(?:_if)?', 'random_shuffle', 'swap', 'all_of', 'any_of', 'none_of', 'copy(?:_if)?']] ],
+    ['array'         , true , true , R[T['array']] ],
+    ['atomic'        , true , true , R[T['atomic_']] ],
+    ['bitset'        , true , true , R[T['bitset']] ],
+    ['bitset'        , true , true , R[T['bitset']] ],
+    ['cassert'       , false, true , R[F['assert']] ],
+    ['chrono'        , true , true , R[C['chrono::duration', 'chrono::time_point']],],
```

Comment:
No, as their full declaration is std::chrono::time_point and std::chrono::duration